db/migrate/20230503003435_add_user_to_post.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToPost < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
+  end
```

Comment:
- [ ] The associations between models must be set up correctly according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).  In order to show that, the  `'author_id'` reference the User class. Kindly write the query like this so the foreign key name will be `author_id`:

```ruby
add_reference :posts, :author, null: false, foreign_key: { to_table: :users }
```

- Note: As you can see on the ERD diagram, they are using the `author_id` column as a foreign key referring to the User class and not the `user_id` column like you did, so that's why we need to build our migration to refer to the author_id column.